R/measures.R

```diff
@@ -396,7 +396,7 @@ rmsle = makeMeasure(id = "rmsle", minimize = TRUE, best = 0, worst = Inf,
 #' @rdname measures
 #' @format none
 mmce = makeMeasure(id = "mmce", minimize = TRUE, best = 0, worst = 1,
-  properties = c("classif", "classif.multi", "req.pred", "req.truth"),
+  properties = c("classif", "classif.multi", "req.pred", "req.truth", "fdaclassif"),
```

Comment:
why as the last property? And in regression as the first property? A bit confusing

I would go from least special to more special. classif -> classif.multi -> fdaclassif
and regr -> fdaregr